notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### :sum -- Show or change summing for 2D images.\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Other implemented/to-implement SPLOT functionality"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Options: histogram\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Option \"histogram\"  plots  the  spectra  in  a  histogram  style\n",
+    "        rather  than  connecting  the  pixel  centers. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Specviz Solution\n",
+    "\n",
+    "`specviz` should default to histogram style, and it's a \"nice to have\" option to be able to toggle.  But either way this is non-critical."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## : log, nolog -- Enable/Disable  logging  of  measurements  to  the file specified by the parameter save_file.\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Options: save_file\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "save_file\n",
+    "        The  file  to  contain  any  results generated by the equivalent\n",
+    "        width or deblending functions.  Results are added to  this  file\n",
+    "        until  the  file is deleted.  If the filename is null (\"\"), then\n",
+    "        no results are saved.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### :# -- Add comment to logfile\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### :show -- Page the full output of  the  previous  deblend  and  equivalent         width measurements.\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Specviz Solution\n",
+    "\n",
+    "\n",
+    "`specviz` will meet the logging need in two ways:\n",
+    "\n",
+    "1. For notebook-oriented workflows, the notebook itself can the \"save file\" - e.g., computing the equivalent width using a `SpectrumRegion` that's explicitly specified already records the result for reproducibility. [ALREADY IMPLEMENTED]\n",
+    "2. For desktop tool workflows (or notebook workflows in part using plugins), a log should be available to track operations. This could/should use the standard Python logging functionality, but to reproduce the specific needs of SPLOT, plugins should have a way to explicitly say \"log my results\". This log file should be accessible in the UI which would then have a \"save\" option for the user to record this if they want to."
+   ]
+  },
```

Comment:
The second type of logging is a 'nice-to-have'.  I can imagine that such a log might become very messy with unwanted commands, so it would be important to be able to clear and start/stop logging at any point.

Reply:
👍 clarified in notebook